### DISCUSSION TITLE
盲审模式的封面包含学校代码

### DISCUSSION THREAD

**Discussion by Yang-Peixin** [nju-lug/NJUThesis#251] [Q&A] [answered]
如题，开启盲审之后，论文封面还有学校代码。请问这个是不是不符合预期啊？
![WechatIMG26785](https://github.com/nju-lug/NJUThesis/assets/44195286/279db173-0f4a-47c7-84a2-1ea443bc54f1)

**Comment by atxy-blip** [comment]
看起来是 #185 的图。原帖给的模板是旧版的设计，没有左上角这块。去年新模板加上了这个小表格，考虑到给的参考范本中能出现校名校徽，似乎学校代码也顺理成章不用隐藏了。

当然，我没找到关于这个的详细规定，如果咨询研究生院或学院教务后有其他说法，欢迎指正。

**Reply by Yang-Peixin** [reply]
我使用的[v1.4.0](https://github.com/nju-lug/NJUThesis/releases/tag/v1.4.0)版本，摘要部分没有学校，封面左上角有学校代码。而且很抱歉，我没有找到“参考范本”，所以认为盲审模式不应该体现学校信息。

我的疑问是，怎么配置可以不显示学校代码呢？为了盲审，我目前做法是在def中删除了code字段。
![WechatIMG436](https://github.com/nju-lug/NJUThesis/assets/44195286/18785032-ff83-4f01-bb28-b4a4fe69dbe2)

**Reply by atxy-blip** [reply, marked as answer]
> 怎么配置可以不显示学校代码呢

事实上有一个专门的学校代码设置项，一般不用修改，所以只在手册里进行说明。赋值为空即可。
```tex
\njusetup{info/school-code={}}
```

<img width="419" alt="image" src="https://github.com/nju-lug/NJUThesis/assets/63113546/92302818-22e7-4284-82d9-51e1f64aafe1">
